learn/on-premise/upgrade/docker-upgrade.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Docker Upgrade"
+id: ""
+sidebar_label: "Docker Upgrade"
+---
+---
+
+## Docker Upgrade
+
+The minimum docker version required for WaveMaker is docker 19.03.13, upgrade docker to the latest version on different types of operating systems follow the below steps.
```

Comment:
The minimum docker version required for WaveMaker is docker 19.03.13. To Upgrade docker to the latest version on different types of operating systems follow the below steps.